tests/e2e/scenarios/test_reborn_webui_v2_notifications.py

```
@@ -0,0 +1,119 @@
+                        "thread_id": THREAD_ID,
+                        "status": "running",
+                        "fired_at": "2026-06-30T08:10:00Z",
+                        "submitted_at": "2026-06-30T08:10:01Z",
+                    }
+                ],
+            }
+        ],
+    }
+
+
+async def _route_automations(page):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_route_automations`

Add return type annotation: `None`

(ANN202)